bin/git-authors

```diff
@@ -43,12 +23,47 @@ authors() {
   fi
 }
 
-#
-# authors.
-#
+if [[ -z "$file" ]]; then
+  file=$(find . -mindepth 1 -maxdepth 1 \( -iname '*authors*' -o -iname '*contributors*' \) | head -n1)
+  if [[ -z "$file" ]]; then
+    file="AUTHORS"
+  fi
+fi
+
+if [[ $# = 0 ]]; then
+  authors >> "$file"
+  exit 0
+fi
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -l|--list )
+      list=1
+      ;;
+    --no-email )
+      no_email=1
+      ;;
+    --output )
+      if [[ -n $2 ]] && [[ $2 =~ ^(\.\/|\.\.\/|\/)?([a-zA-Z_](\/)?)+$ ]]; then
+        output=$2
```

Comment:
I notice the variables `file` and `output` behave and mean the same thing. Is it possible to be consistent and use the old, `file` variable instead of making a new one?